spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/ReactiveResilience4JCircuitBreakerFactory.java

```diff
@@ -103,8 +103,10 @@ public ReactiveCircuitBreaker create(String id, String groupName) {
 			.circuitBreakerConfig(circuitBreakerConfig)
 			.timeLimiterConfig(timeLimiterConfig)
 			.build();
+		boolean isDisableTimeLimiter = ConfigurationPropertiesUtils
+			.isDisableTimeLimiter(this.resilience4JConfigurationProperties, id, groupName);
 		return new ReactiveResilience4JCircuitBreaker(id, groupName, config, circuitBreakerRegistry,
-				timeLimiterRegistry, Optional.ofNullable(circuitBreakerCustomizers.get(id)), isDisableTimeLimiter());
+				timeLimiterRegistry, Optional.ofNullable(circuitBreakerCustomizers.get(id)), isDisableTimeLimiter);
 	}
 
 	private boolean isDisableTimeLimiter() {
```

Comment:
This method is no longer used right?  Maybe we can keep it and call the new Util method or just remove it

Reply:
yes, it's not used. It was private, so just I removed it.